app/containers/FileTree.js

```diff
@@ -35,7 +35,7 @@ function buildTree(folderName, files, handleClick, forceOpen = false) {
         return <File key={value} name={filename} path={value} handleClick={handleClick} />;
       }
       return <UnsupportedFile key={value} name={filename} />;
-    }).valueSeq();
+    }).valueSeq().toArray();
```

Comment:
Gotcha; without using react-immutable-proptypes, writing a PropTypes for an Immutable.List of elements would be un-fun.